vllm/assets/monitors/latency.json

```diff
@@ -0,0 +1,33 @@
+{
+  "version": 2,
+  "created_at": "2024-07-02",
+  "last_updated_at": "2024-07-02",
+  "title": "Average Request Latency is High",
+  "description": "This monitor will alert you if vLLM request latency is too high. High latency means requests are waiting long to be processed. This results in clients having to wait longer for their requests to complete. It also indicates your vLLM server is receiving more requests than it can comfortably handle.",
```

Comment:
```suggestion
  "description": "This monitor alerts you if vLLM request latency is too high. High latency means requests are waiting long to be processed. This results in clients having to wait longer for their requests to complete. It also indicates your vLLM server is receiving more requests than it can comfortably handle.",
```